dist/bootstrap-slider.js

```diff
@@ -1376,8 +1376,8 @@ function _typeof(obj) { return obj && typeof Symbol !== "undefined" && obj.const
 				var val = this._calculateValue(true);
```

Comment:
Wow, I totally missed on this one.

I was using dandrel's comment fix from #392 along with my new code.

Sorry about that guys, a bit new to github and lost track of myself making that change.